src/components/edit/StyledTooltip.tsx

```diff
@@ -0,0 +1,29 @@
+import {
+    styled,
+    Theme,
+    Tooltip,
+    tooltipClasses,
+    TooltipProps,
+} from "@mui/material";
+import { StandardCSSProperties } from "@mui/system";
+import React from "react";
+
+export type TooltipStylesInput = (theme: Theme) => StandardCSSProperties;
+
+// crazy nonsense: https://mui.com/material-ui/react-tooltip/#customization
+// tl;dr, tooltip needs to be styled in a very particular way that overrides its own classname
+// and also targets its child elements. it's very easy to miss so making it a utility
+export const makeStyledTooltipMenu = (styleInput: TooltipStylesInput) => {
+    return styled(({ className, ...props }: TooltipProps) => (
+        <Tooltip {...props} classes={{ popper: className }} />
+    ))(({ theme }) => {
+        const styles = styleInput(theme);
+
+        return {
+            // this tooltip isn't meant to paint over any other surfaces
+            // but extend as an adjacent menu to the line, so z index is 0
+            zIndex: 0,
```

Comment:
snuck this fix in there as well
https://github.com/veedubyou/chord-paper-fe/pull/318 and https://github.com/veedubyou/chord-paper-fe/pull/319 were trying to address this

but upon reflection, perhaps the correct mental model is that this isn't a real "tooltip" and doesn't need to paint over anything else. reducing its z index instead of boosting the player's z index may be the correct move instead